### DISCUSSION TITLE
@types/gulp-htmlhint

### DISCUSSION THREAD

**Discussion by Dulux-Oz** [DefinitelyTyped/DefinitelyTyped#58159] [Request a new @types package] [answered]
Hi All,

Could someone help a (TypeScript) noob (again :-) ) with some advice, please, by answering one (or more) of the following questions:

1. Does anyone have the TypeScript def file for gulp-htmlhint, and may I please have a copy?
2. Could someone please write the TypeScript def file for gulp-htmlhint?
3. Could someone help me to write the TypeScript def file for gulp-htmlhint?
4. Could someone point me towards a different HTML linter that works with Gulp and which is TypeScript-useable?
5. Could someone point me towards how to use gulp-htmlhint "as it is" with TypeScript?

Thanks in advance (& thanks again for the previous help I've been given - I really do appreciate it),

Dulux-Oz

**Comment by martin-badin** [comment, marked as answer]
PR is ready #58182. But first, we have to merge two dependent PR
#58179
#58181

**Comment by Dulux-Oz** [comment]
Thanks Martin  :-)
